lms/djangoapps/commerce/utils.py

```diff
@@ -46,8 +46,7 @@ def __init__(self):
 
     def is_enabled(self, user):
         """ Check if the user is activated, if the service is enabled and that the site is not a microsite. """
-        return (user.is_active and self.config.checkout_on_ecommerce_service and not
-                helpers.is_request_in_themed_site())
+        return user.is_active and self.config.checkout_on_ecommerce_service
```

Comment:
@mattdrayer Right, I missed that `commerce` config is per-instance as well, not per-microsite. I believe the best course of actions would be to push for model-backed site configuration, but if it's not something we could achieve within the time bounds we have (i.e. before the WL-analytics launch), adding site_id would be an appropriate solution.

Alternatively, this setting can live in microsite config as a separate flag ("USE_OTTO") or as an implicit switch (i.e. if `ECOMMERCE_PUBLIC_URL_ROOT` is set).